app/views/projects/index.html.erb

```diff
@@ -37,9 +37,24 @@ See COPYRIGHT and LICENSE files for more details.
     current_user:
   ) %>
 
-  <%= render Projects::FiltersComponent.new(
+  <%= render (Projects::ProjectsFiltersComponent.new(
     query: @query,
-    ) %>
+    ) ) do |component|
+
+    if current_user.allowed_globally?(:add_project)
+      component.with_button( 
+      tag: :a,
```

Comment:
Please check the indentation and lint checks in this file. At least this block seems not intended correctly, and I assume that the space between the paranthesis above is also marked by a linter